.flake8

```diff
@@ -2,4 +2,6 @@
 max-line-length = 160
 extend-ignore =
     # E402 module level import not at top of file
-    E402
\ No newline at end of file
+    E402
+per-file-ignores =
```

Comment:
Add a comment to explain why you are ignoring this

Reply:
Flake8 config updated and comments added to it and elsewhere we are ignoring linting errors in-line code, comments added there as well.